sg/makefile.go

```diff
@@ -42,7 +42,7 @@ func generateMakefile(g *codegen.File, pkg *doc.Package, mk Makefile, mks ...Mak
 	g.P()
 	g.P(".PHONY: clean-sage")
 	g.P("clean-sage:")
-	g.P("\t@git clean -fdx ", filepath.Join(includePath, ToolsDir))
+	g.P("\t@git clean -fdx ", includePath)
```

Comment:
this now removes the bin dir also right? 

Then @sousandrei can remove his fix for that in #133

Reply:
opsie

Reply:
rebased :smile_cat: